### ISSUE TITLE
Getting BIG_CREATION when running the debugger option in intellij

### ISSUE THREAD

**Issue by katebushgg** [KronicDeth/intellij-elixir#1784]
**Describe the bug**
Currently, when I am attempting to run the debugger on Intellij for an Elixir app via IEX Mix, 
I encounter a 

```19:21:54.597 [error] ** :"debugged76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1": Connection attempt from node :"debugger76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**```

**To Reproduce**
1. Go to the debugger config 
2. Set up a debug configuration with a module within another module (pog) is a module within another module (exodia)
3. Click the debug button

**Expected behavior**
BIG_CREATION gets thrown out in the debugger.

**Screenshots**
![image](https://user-images.githubusercontent.com/65788061/83168229-c9f5dc00-a0d6-11ea-9dbf-298219a1ce02.png)

**Logs**
```
cd /Users/stuff/code/<REDACTED>/
export NVM_DIR=/Users/stuff/.nvm
export TMPDIR=/var/folders/rv/l260n59n5x98rh47nbb1dqkm0000gn/T/
export TERM=xterm-256color
export NVM_CD_FLAGS=
export PWD=/Users/stuff/code/exodia/apps/pog
export SHELL=/bin/bash
export USER=stuff
export XPC_SERVICE_NAME=com.jetbrains.intellij.4940
export HOME=/Users/stuff
export SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.3lBJyDamBP/Listeners
export LC_CTYPE=en_US.UTF-8
export LOGNAME=stuff
export VERSIONER_PYTHON_VERSION=2.7
export NVM_BIN=/Users/stuff/.nvm/versions/node/v14.3.0/bin
export NVM_INC=/Users/stuff/.nvm/versions/node/v14.3.0/include/node
export PATH=/Users/stuff/.nvm/versions/node/v14.3.0/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/path/to/elixir/bin
export __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x0
export XPC_FLAGS=0x0
/usr/local/Cellar/erlang/23.0/lib/erlang/bin/erl -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/os_mon-2.5.2/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/diameter-2.2.3/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/tftp-1.0.2/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/ssl-10.0/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/snmp-5.6/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/eunit-2.5/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/runtime_tools-1.15/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/wx-1.9.1/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/mnesia-4.17/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/erts-11.0/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/erl_interface-4.0/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/ssh-4.10/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/crypto-4.7/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/public_key-1.8/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/edoc-0.12/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/erl_docgen-1.0/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/megaco-3.19/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/inets-7.2/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/ftp-1.0.4/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/hipe-4.0/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/parsetools-2.2/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/et-1.6.4/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/debugger-5.0/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/tools-3.4/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/kernel-7.0/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/reltool-0.8/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/eldap-1.2.8/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/compiler-7.6/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/dialyzer-4.2/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/asn1-5.0.13/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/observer-2.9.4/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/xmerl-1.3.25/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/syntax_tools-2.3/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/stdlib-3.13/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/common_test-1.19/ebin -pa /usr/local/Cellar/erlang/23.0/lib/erlang/lib/sasl-4.0/ebin -pa /usr/local/Cellar/elixir/1.10.3/lib/logger/ebin -pa /usr/local/Cellar/elixir/1.10.3/lib/elixir/ebin -pa /usr/local/Cellar/elixir/1.10.3/lib/ex_unit/ebin -pa /usr/local/Cellar/elixir/1.10.3/lib/mix/ebin -pa /usr/local/Cellar/elixir/1.10.3/lib/eex/ebin -pa /usr/local/Cellar/elixir/1.10.3/lib/iex/ebin -elixir ansi_enabled true -name debugged76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1 -setcookie 76537fa8-d8a3-4865-98e9-52d4bca4417c -eval application:ensure_all_started(elixir) -eval 'Elixir.Code':require_file(<<\"/private/var/folders/rv/l260n59n5x98rh47nbb1dqkm0000gn/T/intellij_elixir125/debugger/lib/intellij_elixir/debugger/server.ex\">>) -eval 'Elixir.Code':require_file(<<\"/private/var/folders/rv/l260n59n5x98rh47nbb1dqkm0000gn/T/intellij_elixir125/debugger/lib/intellij_elixir/debugged.ex\">>) -eval 'Elixir.Code':require_file(<<\"/private/var/folders/rv/l260n59n5x98rh47nbb1dqkm0000gn/T/intellij_elixir125/debugger/lib/mix/tasks/intellij_elixir/debug.ex\">>) -noshell -user Elixir.IEx.CLI -extra --no-halt -S /usr/local/Cellar/elixir/1.10.3/bin/mix do intellij_elixir.debug, run
['debugged76537fa8-d8a3-4865-98e9-52d2020-05-27 19:21:53.556833 
    args: 4bca4417c@127.0.0.1',"from",'debugger76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1',"rejected",["BIG_CREATION"]]
    format: "** ~w: Connection attempt ~s node ~w ~s since it cannot handle ~p.**~n"
{    label: error_logger,error_msg}
Erlang/OTP 23 [erts-11.0] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe] [dtrace]

=ERROR REPORT==== 27-May-2020::19:21:53.660486 ===
** 'debugged76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1': Connection attempt from node 'debugger76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1' rejected since it cannot handle ["BIG_CREATION"].**

=ERROR REPORT==== 27-May-2020::19:21:53.556833 ===
** 'debugged76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1': Connection attempt from node 'debugger76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1' rejected since it cannot handle ["BIG_CREATION"].**

=ERROR REPORT==== 27-May-2020::19:21:53.769131 ===
** 'debugged76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1': Connection attempt from node 'debugger76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1' rejected since it cannot handle ["BIG_CREATION"].**


19:21:53.902 [error] ** :"debugged76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1": Connection attempt from node :"debugger76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**

 
19:21:53.977 [error] ** :"debugged76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1": Connection attempt from node :"debugger76537fa8-d8a3-4865-98e9-52d4bca4417c@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**
```

**Desktop:**
 - OS: [e.g. macOS]
 - Version [e.g. 10.14.3]

**Erlang:**
- Installer [e.g. homebrew]
- Version [e.g. 21]

**Elixir:**
- Installer [e.g. kiex]
- Version [e.g. 1.8]

**Plugin:**
- Version [e.g. 10.3.0]

**Additional context**

**Comment by praburajan** [comment]
I am seeing the same issue. Appreciate a reply

**Comment by KronicDeth** [comment]
A way to tell if it is a JInterface compatibility issue is to see if the issue goes away if you downgrade the version of Erlang.

**Comment by praburajan** [comment]
This is the output of erl --version

`erl --version                                                                                                      
Erlang/OTP 23 [erts-11.0] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe] [dtrace]

Eshell V11.0  (abort with ^G)`

**Comment by praburajan** [comment]
> A way to tell if it is a JInterface compatibility issue is to see if the issue goes away if you downgrade the version of Erlang.



> A way to tell if it is a JInterface compatibility issue is to see if the issue goes away if you downgrade the version of Erlang.

What version of erlang do you want me to try to downgrade?

**Comment by KronicDeth** [comment]
You’re on 23, so try 22 something.  It has to be something in the newer versions of OTP since it’s happening now and not earlier for others.

**Comment by praburajan** [comment]
I have downgraded to Erlang/OTP 22 and still facing the same issue

elixir -v
Erlang/OTP 22 [erts-10.7.2] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.10.3 (compiled with Erlang/OTP 22)

Connection attempt from node :"debugger28af1a0b-7f49-4a69-bd03-adddf0835438@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**

**Comment by KrishManohar** [comment]
I've tested it on Elixir 1.10.3 (compiled with Erlang/OTP 23) and it's giving same error.

**Comment by Fabsolute** [comment]
```
2020-06-27 15:08:20.063458 
    args: ['debugged4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1',"from",'debugger4e211ba9-a280-40e1-abaf-66de3140a    format: 2d5@127.0.0.1',"rejected",["BIG_CREATION"]]
"** ~w: Connection attempt ~s node ~w ~s since it canno    label: t handle ~p.**~n"
{error_logger,error_msg}
=ERROR REPORT==== 27-Jun-2020::15:08:20.063458 ===
** 'debugged4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1': Connection attempt from node 'debugger4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1' rejected since it cannot handle ["BIG_CREATION"].**

=ERROR REPORT==== 27-Jun-2020::15:08:20.165201 ===
** 'debugged4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1': Connection attempt from node 'debugger4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1' rejected since it cannot handle ["BIG_CREATION"].**


15:08:20.292 [error] ** :"debugged4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1": Connection attempt from node :"debugger4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**

 
15:08:20.369 [error] ** :"debugged4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1": Connection attempt from node :"debugger4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**

 
15:08:20.470 [error] ** :"debugged4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1": Connection attempt from node :"debugger4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**

 
15:08:20.571 [error] ** :"debugged4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1": Connection attempt from node :"debugger4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**

 
15:08:20.674 [error] ** :"debugged4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1": Connection attempt from node :"debugger4e211ba9-a280-40e1-abaf-66de3140a2d5@127.0.0.1" rejected since it cannot handle ['BIG_CREATION'].**

** (Mix) The task "intellij_elixir.debug" could not be found because the module is named Mix.Tasks.IntellijElixir.Debug instead of Mix.Tasks.IntellijElixir.Debug as expected. Please rename it and try again

```

I am having same issue too

**Comment by koudelka** [comment]
As a note, I was having this same issue attempting to use JInterface with an unrelated project.

Compiling the jar from the [otp repository](https://github.com/erlang/otp) itself fixed it. Not exactly ideal, since it bypasses the package manager, but it works.

Probably just need to bug the core team to publish a new version.

**Comment by austinarbor** [comment]
@koudelka can you please clarify your fix? I tried compiling erlang from source and swapped the `jinterface-1.6.1.jar` in the plugin directory with the compiled `OtpErlang.jar` (which I renamed to jinterface-1.6.1) , but that did not resolve the issue for me

**Comment by KronicDeth** [comment]
Found https://bugs.erlang.org/browse/ERL-928.  It turns out that the OTP team wasn't maintaining the Maven published version of JInterface, it was Basho, and since Basho is gone the Maven repos won't be updated anymore unless someone else takes it on, so I could do it, but only so that I could consume it.  Seems I should just pull the jar from the locally installed version of Erlang from now now on.  I think I was doing that before I found the Maven version anyway or maybe I was vendoring it?
